draftfast/rules.py

```diff
@@ -131,6 +133,10 @@ def get_nfl_showdown_positions(dk: bool = False, fd: bool = False) -> list:
         'NHL_SHOWDOWN': [
             ['FLEX', 6, 6],
         ],
+        'MLB_SHOWDOWN': [
+            ['CPT', 1, 1],
```

Comment:
This is an issue I think for all the other showdown games. It at least impacts the display if `CPT` isn't a constraint here - everything shows as `FLEX`.

Reply:
yeah, i think NHL showdowns are broken or at least non-optimal right now. Look at `get_nfl_showdown_positions`  or `NBA_SHOWDOWN` rules